chain/network/src/routing/graph_v2/mod.rs

```diff
@@ -554,9 +556,12 @@ impl Inner {
         distances: HashMap<PeerId, u32>,
     ) -> Option<network_protocol::DistanceVector> {
         if self.my_distances == distances {
+            tracing::debug!(target: "routing", "No change to routing distances after processing network updates");
             return None;
         }
 
+        tracing::debug!(target: "routing", "Routing distances have changed; reconstucting distance vector");
```

Comment:
is it worth noting what the actual change was in the logging?

Reply:
It is logged later; we print the complete state after we finish reconstructing it.